src/ontology/ro-edit.owl

```diff
@@ -6523,6 +6525,28 @@ SubObjectPropertyOf(obo:RO_0017004 obo:RO_0002610)
 SymmetricObjectProperty(obo:RO_0017004)
 IrreflexiveObjectProperty(obo:RO_0017004)
 
+# Object Property: obo:RO_0017500 (has life cycle stage)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0017500 "A relationship between an organism and a stage it undergoes within its full life cycle."@en)
+AnnotationAssertion(dc:contributor obo:RO_0017500 <https://orcid.org/0000-0002-1373-1705>)
+AnnotationAssertion(dc:contributor obo:RO_0017500 <https://orcid.org/0000-0002-8844-9165>)
+AnnotationAssertion(dc:date obo:RO_0017500 "2023-08-09T18:37:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment obo:RO_0017500 "This relation accommodates social, behavioural and biological (developmental) stages."@en)
+AnnotationAssertion(rdfs:label obo:RO_0017500 "has life cycle stage"@en)
```

Comment:
please add example usages for both

Reply:
Resolved for this relation; "collected during" relation how handled in separate pull request.

Reply:
Great, thanks for updating this.